pom.xml

```diff
@@ -0,0 +1,198 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sonatype.oss</groupId>
+    <artifactId>oss-parent</artifactId>
+    <version>7</version>
+  </parent>
+  <groupId>org.duraspace</groupId>
+  <artifactId>codestyle</artifactId>
+  <version>1.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>codestyle</name>
+  <description>A package that contains checkstyle rules for DuraSpace projects</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+  </properties>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>2.6.1</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.2</version>
+          <configuration>
+            <debug>false</debug>
+            <source>1.7</source>
+            <target>1.7</target>
```

Comment:
Just a note here that it's slightly odd to have a Java 1.7 requirement here. But, this might be required to generate a JAR?

As a sidenote: Assuming this is necessary, we'll have to be careful not to set this Java requirement too high, as it will need to be compatible with all projects. For example, if we set this at 1.8, it might not be compatible with projects requiring/using Java 1.7.  So, 1.7 is probably the correct setting for now.

Reply:
I removed the `maven-compiler-plugin` locally and built. The result is that the default `maven-compiler-plugin` and configuration is used... which resulted in a Java8 artifact.

Reply:
Thanks for that test. In that case, I suggest leaving this as-is. We likely want this to be explicit in the POM so there are no future surprises.  I'm not sure if all projects are completely Java 8 yet or not, so leaving this as Java 7 is fine by me.